pwa/src/templates/templateParts/formHeader/FormHeaderTemplate.tsx

```diff
@@ -5,6 +5,7 @@ import { t } from "i18next";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { Heading1, Button } from "@gemeente-denhaag/components-react";
 import { faFloppyDisk, faMinus, faPlus, faTrash } from "@fortawesome/free-solid-svg-icons";
+import clsx from "clsx";
```

Comment:
(See other comments)

Since we don't need to have a second class name anywhere with the suggested changes, this import will become redundant.